### PR TITLE
Allow stream_channel version 2.x

### PR DESCRIPTION
**Pull request by natebosch**
Correct style on other constraints.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,18 +1,17 @@
 name: shelf_web_socket
-version: 0.2.2+4
+version: 0.2.2+5
 
 description: A WebSocket handler for Shelf.
 author: Dart Team <misc@dartlang.org>
 homepage: http://github.com/dart-lang/shelf_web_socket
```
Comment (kevmoo) [inline pubspec.yaml]: while you're here: would you make this https?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,18 +1,17 @@
 name: shelf_web_socket
-version: 0.2.2+4
+version: 0.2.2+5
 
 description: A WebSocket handler for Shelf.
```
Comment (kevmoo) [inline pubspec.yaml]: Could you also make this a bit longer? See https://pub.dartlang.org/packages/shelf_web_socket#-analysis-tab-

File: pubspec.yaml (hunk 1 above)
Comment (natebosch) [reply pubspec.yaml]: Sure

File: pubspec.yaml (hunk 2 above)
Comment (natebosch) [reply pubspec.yaml]: I guess? Seem arbitrary.

Comment (kevmoo) [reply pubspec.yaml]: Optimizing for Google SEO

**Pull request merged**